external/common/lapack/FindMathOpenMP.cmake

```diff
@@ -101,16 +101,15 @@ endif()
 
 if (ENABLE_OPENMP)
     # *not* REQUIRED b/c some compilers don't support OpenMP and -DENABLE_OPENMP isn't a build-or-die-trying
-    find_package(TargetOpenMP QUIET COMPONENTS ${TargetOpenMP_FIND_COMPONENTS})
+    find_package(TargetOpenMP COMPONENTS ${TargetOpenMP_FIND_COMPONENTS})
+    set(PN MathOpenMP)
+    if (TargetOpenMP_FOUND)
+        set_property(TARGET tgt::${PN} APPEND PROPERTY INTERFACE_LINK_LIBRARIES OpenMP::OpenMP)
+    else()
+        message(WARNING "${PN} configuration failed! The code will be built without OpenMP.")
+    endif()
 endif()
 
-set(PN MathOpenMP)
-get_property (_ill TARGET tgt::MathOpenMP PROPERTY INTERFACE_LINK_LIBRARIES)
-set (${PN}_MESSAGE "Found MathOpenMP: ${_ill}")
-    
+set(${PN}_FOUND 1)
 include(FindPackageHandleStandardArgs)
-find_package_handle_standard_args (${PN} DEFAULT_MSG ${PN}_MESSAGE)
-
-if (TARGET OpenMP::OpenMP)
-    set_property(TARGET tgt::MathOpenMP APPEND PROPERTY INTERFACE_LINK_LIBRARIES "OpenMP::OpenMP")
-endif()
+find_package_handle_standard_args (${PN} DEFAULT_MSG ${PN}_FOUND)
```

Comment:
This should be: `find_package_handle_standard_args(${PN} DEFAULT_MSG ${${PN}_LIBRARIES})`

Reply:
@robertodr `${${PN}_LIBRARIES}` is only defined if building with MKL